src/geo/ui/legends/custom-choropleth/legend-template.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="u-flex u-justifySpace u-bSpace--m">
+  <p class="CDB-Text CDB-Size-small"><%- prefix %> <%- suffix %></p>
```

Comment:
For heatmaps, we don't know the values, at least it's not trivial to get them, so we are generating only the ramp of colors.